logs/2020-03-03 21:12:33.422273 demo.csv

```diff
@@ -0,0 +1 @@
+"2020-03-03 21:12:33.427964","Experiment demo started"
```

Comment:
Hmm this shouldn't be here. Is one of the unit tests creating this?

Reply:
Empty.py shouldn't be in the project either.

Reply:
Same with unittest.csv

Reply:
These are still there from before they were being .gitignore'd, removed them